server/src/services/createCode/createCode.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/extensions */
+import { EXTERNAL_SERVER_URL } from '../../config/trulioo.config.js';
+import { createNewCode } from '../../helpers/codeUtils.js';
+
+export const createCode = async (request, response) => {
+  try {
+    console.log('Creating a new URL');
+    const expiryDate = new Date();
+    expiryDate.setDate(expiryDate.getDate() + 365);
```

Comment:
Instead of 365, let's use an env var, and let's find out from FLO team what the default should be. My guess is something shorter, like 7 or 30.

Reply:
In our local dev env we'll be able to set it to 365 or whatever long value.